fetch/build.sbt

```diff
@@ -0,0 +1,73 @@
+import EndpointsSettings._
+
+val `fetch-client` =
+  project
+    .in(file("client"))
+    .enablePlugins(ScalaJSPlugin)
+    .configure(_.disablePlugins(ScoverageSbtPlugin))
+    .settings(
+      publishSettings,
+      `scala 2.12 to 2.13`,
+      name := "fetch-client",
+      version := "3.1.0+n",
+      versionPolicyIntention := Compatibility.None,
+      //disable coverage for scala.js: https://github.com/scoverage/scalac-scoverage-plugin/issues/196
+      coverageEnabled := false,
+      libraryDependencies ++= Seq(
+        "org.scala-js" %%% "scalajs-dom" % "1.2.0",
```

Comment:
Should we use version 2.0.0?

Reply:
We can, I do not think that using the latest version available could cause issues.

Reply:
This also must be done for Scala 3 support.